docs/using-radicle/pushing-changes.md

```diff
@@ -0,0 +1,46 @@
+---
+id: pushing-changes
+title: Pushing changes
+---
+
+To push code to Radicle, you first need to make sure your shell is configured
+correctly by following the instructions in the [Getting Started][gs] section to
```

Comment:
Is there a link to the header for these instructions rather than the whole file?

Reply:
Yes, can do